backend/migrations/00072_add_facility_to_learning_record_entries.sql

```
@@ -0,0 +1,17 @@
+-- +goose Up
+-- +goose StatementBegin
+ALTER TABLE public.learning_record_entries
+    ADD COLUMN facility_id    INTEGER REFERENCES public.facilities(id) ON DELETE SET NULL,
+    ADD COLUMN facility_other TEXT NOT NULL DEFAULT '';
+
+CREATE INDEX IF NOT EXISTS idx_learning_record_entries_facility_id ON public.learning_record_entries(facility_id);
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)